week2/ShoppingList/cart.js

```diff
@@ -0,0 +1,289 @@
+const ITEM_LIST_KEY = "itemList";
+//장바구니(cart)페이지 구현 - 로컬 스토리지 활용
+const cartBody = document.getElementById('cartBody');
+
+const storageItems = localStorage.getItem(ITEM_LIST_KEY);
+const parsedItems = JSON.parse(storageItems); //localStorage에서 가져온 아이템
```

Comment:
이 부분은 한 번에 작성할 수 있어요 !
```suggestion
const storageItems = JSON.parse(localStorage.getItem(ITEM_LIST_KEY));
```
만약 storage에 없는 경우도 대비해서 코드를 작성해주면 훨씬 안전한 코드가 되겠죠 ?
아마 코드리뷰를 돌아다니면서 많이 마주했을거라고 생각해서 따로 작성은 안해둘게요 ! 리팩토링때 적용해보아요 :)